### ISSUE TITLE
[Feature Request] Optional forced logging enabled

### ISSUE THREAD

**Issue by MinerSpark** [vrcx-team/VRCX#812]
![image](https://github.com/vrcx-team/VRCX/assets/22899721/15e32394-a492-4086-b61b-a9cd7d8228c7)

Allow the option to toggle the auto enable of logging.
This is quite irritating for running more then 1 game for example world testing without using the offical build and test.
I myself am running 4 games and having one user join the world logged for 4 times is quite irritating so disabling logging for all but 1 client is required.

**Comment by Nekromateion** [comment]
Hey,

you can achieve the desired effect by turning on the "Disable GameLog" option in the Advanced Settings.
![image](https://github.com/vrcx-team/VRCX/assets/43814053/d2df1acc-5604-4611-8cb2-eccc10d7b7c6)

**Comment by MinerSpark** [comment]
> Hey,
> 
> you can achieve the desired effect by turning on the "Disable GameLog" option in the Advanced Settings. ![image](https://private-user-images.githubusercontent.com/43814053/342114663-d2df1acc-5604-4611-8cb2-eccc10d7b7c6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkyMDc2NDMsIm5iZiI6MTcxOTIwNzM0MywicGF0aCI6Ii80MzgxNDA1My8zNDIxMTQ2NjMtZDJkZjFhY2MtNTYwNC00NjExLThjYjItZWNjYzEwZDdiN2M2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI0VDA1MzU0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkzNTE1OTc1MjhiYzJjY2U1NGUxZjEyZTExZDI3MjBkNjFkNGIyM2FhMTExYTJmZGYyZWRhZTVjNWRiMmFkNjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zGnARfp1vZ2VuffjpgSqc-kw-GKMjn5z5PJNe-5vyNQ)

Yeah but thats force disabling it everytime, So other clients are disabled but my primary client stays logged on, That option would have VRCX auto disable it every launch which is Not what I need, VRCX Option for Controlling the logs needs to be an Optional option, rather then the "Forced" state its currently in.

**Comment by Nekromateion** [comment]
> Yeah but thats force disabling it everytime, So other clients are disabled but my primary client stays logged on, That option would have VRCX auto disable it every launch which is Not what I need, VRCX Option for Controlling the logs needs to be an Optional option, rather then the "Forced" state its currently in.

No, it won't force disable it every time you start up your game if you enable "Disable GameLog". Once it is enabled it will not make any changes to the LOGGING_ENABLED registry key at all. Like I said, enabling it will achieve the exact thing you are talking about in this issue. Unless you are talking about wanting to keep VRCX logs while "Disable GameLog" is enabled, but going off of everything you have said so far it doesn't seem like that is what you were talking about.

**Comment by MinerSpark** [comment]
I cannot close my game to test this right now, But apon next time I close my game I will see.
End of my goal is having VRCX logs still visable e.g join/leave logs without having VRCX changing the reg key at anytime.
Which seems like it could do what I need.

**Comment by Nekromateion** [comment]
Okay, with "Disable GameLog" enabled you will not get any new entries in the Game Log tab but it will not interfere with your VRChat logging option. So this specific behavior that you are looking for is not possible with the current state of VRCX.

**Comment by MinerSpark** [comment]
Yes, Thats exactly why this is a feature request.

**Comment by Natsumi-sama** [comment]
It's too niche that I don't think it's worth making an option for, but here is a script that'll nuke the function to achieve what you're after.
`%AppData%\VRCX\custom.js`
```
$app.checkVRChatDebugLogging = () => {};
```
